.github/workflows/generate-component-labels.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Generate component labels'
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
```

Comment:
Is this line needed?

Reply:
Oh nvm. I see it's needed for manual triggering

Reply:
I was going to leave the option to do a manual trigger, but I think the push conditions now are sufficient to remove the need for a manual option.

Reply:
On second thought, I added it back. It really doesn't hurt to have in there for an admin function like this.